.vale.ini

```diff
@@ -0,0 +1,22 @@
+; Path to the "vale" styles
+StylesPath = .github/styles
```

Comment:
Why use . github for this?

Reply:
The styles that Vale recognises & uses has to be stored somewhere & the [official documentations suggests](https://vale.sh/docs/topics/packages/#packages-and-vcs) keeping it inside the `.github/styles` directory. There's no particular reason & the `styles` directory can be stored anywhere in the repository.

Reply:
I’d say `.vale/styles`. We don’t use `.github` for non-GitHub things, and I don’t think we should make an exception here.

Reply:
Yeah right, `.vale/styles` sounds way less ambiguous. I'll move the styles to that directory instead.